config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3,6 +3,7 @@
 ## Properties files load order: component-manager/kernel -> config/default -> sakai/sakai -> sakai/local -> sakai/security
 # This file is read by the Sakai Kernel and overrides kernel.properties
 # All other sakai.properties files defined locally override these settings
+portal.show.role=true
```

Comment:
portal.show.current.role and please include a description.

# S2U-44 Display the current user role in the portal.
# DEFAULT: true
# portal.show.current.role=false